src/test/scala/org/hammerlab/guacamole/alignment/AffineGapPenaltyAlignmentSuite.scala

```diff
@@ -13,7 +13,7 @@ class AffineGapPenaltyAlignmentSuite extends FunSuite with Matchers {
       openGapProbability = 1e-3,
       closeGapProbability = 1e-2
     )
-    alignments(4)._3.toInt should be(0)
+    alignments(0)._3.toInt should be(0)
```

Comment:
The alignments score matrix is in reverse now, so the full alignment is no longer in the last position, but in the first.